src/cascadia/TerminalSettingsEditor/Actions.idl

```diff
@@ -5,19 +5,53 @@ import "EnumEntry.idl";
 
 namespace Microsoft.Terminal.Settings.Editor
 {
+    runtimeclass RebindKeysEventArgs
+    {
+        Microsoft.Terminal.Control.KeyChord OldKeys { get; };
+        Microsoft.Terminal.Control.KeyChord NewKeys { get; };
+    }
+
+    runtimeclass KeyBindingViewModel : Windows.UI.Xaml.Data.INotifyPropertyChanged
+    {
+        // Settings Model side
+        String Name { get; };
+        String KeyChordText { get; };
+
+        // UI side
+        Boolean ShowEditButton { get; };
+        Boolean IsInEditMode { get; };
+        String ProposedKeys;
+        Windows.UI.Xaml.Controls.Flyout AcceptChangesFlyout;
+        String EditButtonName { get; };
+        String CancelButtonName { get; };
```

Comment:
button names do not seem like the kind of thing that goes in the view model. they belong in the _view_. Why are they here?

Reply:
Ugh. This is the same problem that happened on _all_ the settings UI pages. I want the tooltip and the automation property to be the same value, but I don't want to introduce 2 resources. Normally, I'd reuse the resources by...
1. give the control an `x:Name`
2. in the page's ctor, manually set the automation property

Unfortunately, we can't do that here, because data templates don't let you get anything by name. So I made the view model responsible for getting the tooltip's text (via the resource) and spitting it out.

I tried doing the `RelativeSource` thing, but that didn't work. If we can figure out a way to fix this here (or on any of the SUI pages), we can cut a ton of code.